packages/synapse-react-client/src/components/AccessRequirementList/AccessRequirementList.tsx

```diff
@@ -294,6 +296,14 @@ export default function AccessRequirementList(
     </>
   )
 
+  const dialogWidth =
+    [
+      RequestDataStep.UPDATE_ACCESSORS_AND_FILES,
+      RequestDataStep.UPDATE_RESEARCH_PROJECT,
+    ].includes(requestDataStep) && canShowManagedACTWikiInWizard
+      ? 'xl'
+      : 'md'
```

Comment:
Dialog should be larger if we're on a ManagedACTAR step and the screen size is large enough to show it